### ISSUE TITLE
Segmentation Network

### ISSUE THREAD

**Issue by phinoo** [AlexeyAB/darknet#4583]
hi Alex,Thanks for your great work. Is it possible to implement a segmentation network under Darknet?

**Comment by bagminer** [comment]
Yes its possible. You can find lot of image segmentation implementation in github for reference.

**Comment by LukeAI** [comment]
[Yolact](https://github.com/dbolya/yolact) is darknet-inspired
